src/state/actions/window.js

```diff
@@ -27,6 +27,7 @@ export function addWindow(options) {
     thumbnailNavigationPosition: 'bottom', // bottom by default in settings.js
     xywh: [0, 0, 400, 400],
     rotation: null,
+    view: 'single',
```

Comment:
```suggestion
    viewType: 'single',
```
This is a small suggestion, but we might save ourselves a tiny headache down the line by having this config key match our action types name.

Reply:
If its ok, can we hold off on this name change? I agree its suboptimal right now. Perhaps we can brainstorm to figure out something better for internal/and external usage?